src/shared/components/cancerSummary/SummaryBarGraph.tsx

```diff
@@ -32,24 +33,23 @@ export default class SummaryBarGraph extends React.Component<ISummaryBarGraphPro
         this.updateChart = this.updateChart.bind(this);
     }
 
-    private getTooltipOptions(tooltipModel: any, data:IBarGraphConfigOptions, chartOptions:any) {
-        // $('#chartjs-tooltip').remove();
+    private getTooltipOptions(tooltipModel: any, data:IBarGraphConfigOptions, chartOptions:any, uniqueId:string) {
 
         // Tooltip Element
-        let tooltipEl = document.getElementById('chartjs-tooltip');
+        let tooltipEl = document.getElementById('chartjs-tooltip-' + uniqueId);
```

Comment:
this should really be unique to the feature, not chartjs.  i.e. cancer-type-summary-tab-xxx